specification/ai/Azure.AI.Agents/run_steps/models.tsp

```diff
@@ -435,6 +435,17 @@ model RunStepDeltaMicrosoftFabricToolCall extends RunStepDeltaToolCall {
   microsoftFabric: Record<string>;
 }
 
+@doc("Represents the SharePoint tool call in a streaming run step.")
+@added(Versions.v2025_05_15_preview)
+model RunStepDeltaSharepointToolCall extends RunStepDeltaToolCall {
+  @doc("The object type, which is always 'sharepoint_grounding'")
+  type: "sharepoint_grounding";
+
+  @doc("SharePoint tool input and output.")
+  @encodedName("application/json", "sharepoint_grounding")
+  sharePoint: Record<string>;
```

Comment:
I missed this consideration for `microsoftFabric` above: can we align the name closer to the type?

- Probably later: ``sharepoint_grounding`` can work, including not needing `@encodedName`, and I *believe* doesn't cause warnings anymore (adjusting in this direction can be done later, though)
- For now: `sharepointGrounding` is the closest camel-cased adaptation of the discriminator

If the SDK naming is undesirable, that can/should be adjusted with per-language encodedNames (ideally in language-specific .tsp files)

Reply:
Fixed.